src/lib/holocene/drawer.svelte

```diff
@@ -16,6 +16,10 @@
   export let id = 'navigation-drawer';
   export let closeButtonLabel: string;
   export let closePadding: boolean = true;
+  export let target: 'body' | string = 'body';
```

Comment:
What if I pass `'ross'` as `target` ? Will I get portaled to another dimension? (Is there a better type we can use here?)

Reply:
Gonna be honest, don't remember why I thought I needed this. Removed the prop and always use document.body